docs/accessing-orcd/ssh-setup.md

```diff
@@ -0,0 +1,110 @@
+---
+tags:
+ - SSH Key Setup
+---
+
+# SSH Key Setup
+
+An SSH key is a secure access credential used in the SSH protocol and establishes a secure and encrypted connection to our HPC systems. This page is for those who wish to implement SSH key authentication on top of general MIT Kerberos authentication.
+
+SSH keys consist of a pair: a public key and a private key. 
+
+- **Public Key**: This key can be shared freely and is used to encrypt data that only the corresponding private key can decrypt.
+- **Private Key**: This key must be kept secure and private. It is used to decrypt data encrypted with the corresponding public key and to prove the identity of the user during the SSH authentication process.
+
+When you attempt to connect to an HPC system using SSH key authentication, the system uses your public key to initiate a challenge that can only be answered correctly using your private key. If the correct response is received, the system verifies your identity and grants access. 
+
+Using the key and lock analogy, the private key is like your key, and the public key is like a lock you might place on a gym locker, which would be like your account on the SuperCloud system. You can leave the lock locked on the locker at the gym, because no one can open the lock without the key, but you wouldn't want to share your key with anyone else, because then they could get into your locker.
+
+> **Important:** Your private key should never be shared with anyone. If someone else obtains your private key, they could potentially gain unauthorized access to any system your key is associated with.
```

Comment:
Maybe use the "danger" admonition: https://squidfunk.github.io/mkdocs-material/reference/admonitions/#+type:danger

Markdown would be something like:

```
!!! danger "Do not share your private key"
    Your private key should never be shared with anyone. If someone else obtains your private key, they could potentially gain unauthorized access to any system your key is associated with.
```